doc/source/drivers/raster/pds4.rst

```diff
@@ -421,7 +421,7 @@ its parameterized variables:
 
    $ gdal_translate input.tif output.xml -of PDS4 \
                -co VAR_TARGET_TYPE=Satellite \
-               -co VAR_Target=Moon \
+               -co VAR_TARGET_TYPE=Moon \
```

Comment:
Did you mean "VAR_TARGET_NAME"?

Reply:
I was to quick , you're absolutely right.